### ISSUE TITLE
permission denied

### ISSUE THREAD

**Issue by Git72job** [jlloyola/pcloudcc_docker#4]
When I do a (docker compose up -d) I get this message because it could be

WARN[0000] /home/juan/PCLOUD_DRIVE/compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion
permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.46/containers/json?all=1&filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.project%3Dpcloud_drive%22%3Atrue%7D%7D": dial unix /var/run/docker.sock: connect: permission denied

The only problem is that I have:
UID: 1000
GID: 100. 
is this the reason

**Comment by BlockyBlockling** [comment]
Do you have docker installed?
You can try it with `docker version`
Please give me the output then.

**Comment by BlockyBlockling** [comment]
Also the Problem could be caused if you ran the `docker-compose up -d` command as user without the Proper permissions.
Then you can either use `sudo` or give your User the Permission to use Docker (Add him to the docker group).
https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

**Comment by Git72job** [comment]
they needed permission docker.

I have already started  docker compose up -d  
 ✔ Network pcloud_drive_default        Created                                                                     0.4s
 ✔ Volume "pcloud_drive_pcloud_cache"  Created                                                                     0.0s
 ✔ Container pcloud_drive-pcloud-1     Started 
                                                                                            
but when putting the instruction    (  docker attach pcloudcc_docker-pcloud-1  )   
I get      (   Error response from daemon: No such container: pcloudcc_docker-pcloud-1   )

because it says it doesn't exist I have to access it from the docker folder with root ??

**Comment by BlockyBlockling** [comment]
No its just the wrong container name. The right name for you is pcloud_drive-pcloud-1
So the correct Command would be `docker attach pcloud_drive-pcloud-1`

**Comment by Git72job** [comment]
Sorry for so many questions but I'm new to this.

once you put the command     docker attach pcloud_drive-pcloud-1
it waits for you to hit
Down: Everything Downloaded| Up: Everything Uploaded, status is LOGIN_REQUIRED
Please, enter password
logging in
Down: Everything Downloaded| Up: Everything Uploaded, status is CONNECTING
Down: Everything Downloaded| Up: Everything Uploaded, status is TFA_REQUIRED
Please enter 2fa code
123456
Down: Everything Downloaded| Up: Everything Uploaded, status is CONNECTING
Down: Everything Downloaded| Up: Everything Uploaded, status is SCANNING
Down: Everything Downloaded| Up: Everything Uploaded, status is READY

it's right?

**Comment by BlockyBlockling** [comment]
Yeah should be correct. Just check your folder and try it out. But it looks good.

**Comment by Git72job** [comment]
it's supposed to have worked     - Down: Everything Downloaded| Up: Everything Uploaded, status is READY  -

I have executed the command     -   fusermount -uz   -    with the same directory you specified in the .env 

but in that directory I don't see the Pcloud files.

???????

**Comment by BlockyBlockling** [comment]
So what is set in your .ENV for the Variable PCLOUD_DRIVE?
`/home/user/pCloudDrive` or `/home/juan/pCloudDrive`

**Comment by Git72job** [comment]
PCLOUD_DRIVE=/home/juan/PCLOUD_DRIVE

**Comment by BlockyBlockling** [comment]
Are there any changes on the PCloud Drive like is the compose file in your Drive now?

**Comment by Git72job** [comment]
is he   [compose.yml]  
and the file .env

**Comment by BlockyBlockling** [comment]
Okay so he uploaded the folder but he didnt download? Is there anythin in the Drive but not on the local directory where the compose file is in on the linux host?

**Comment by Git72job** [comment]
I don't quite understand you ???

**Comment by BlockyBlockling** [comment]
So for me it seems that the upload worked cause it syncs the directory where the Compose file is in. So the only thing we need to check is download. The way to check this is to upload a random file via pcloud and then restarting the container to force the sync
`docker container restart pcloud_drive-pcloud-1`

**Comment by Git72job** [comment]
WHEN ACCESSING THE DIRECTORY    /inicio/juan/PCLOUD_DRIVE   con   SAMBA   no encuentro archivos PCLOUD .
when accessing the program  WINSCP with  ROOT  no pcloud files
They even sent me an email from pcloud that a new device has accessed the cloud.
but there is no way I won't do something right.

thanks for everything anyway.

something escapes me

**Comment by Git72job** [comment]
$ cd  PCLOUD_DRIVE
$ ls
 Archivos            Cert.Matr.Paula2024-2025.pdf  'My Pictures'  'Public Folder'
'Automatic Upload'   Menu-Yuso_GRUPOS-35E.pdf      'My Videos'    'System Volume Information'
 Backups            'My Music'                     'New Folder'
$

The PCLOUD folders are there but it won't let me open them from anywhere
How is it possible?????????

**Comment by BlockyBlockling** [comment]
Use ls -l to see who owns them. It should work if you uid and gid of the env file is the Same as the ones you wrote in the First message.

**Comment by Git72job** [comment]
$ ls -lS
total 135
-rw-r--r-- 1 juan users 83464 sep  6 17:55  Menu-Yuso_GRUPOS-35E.pdf
-rw-r--r-- 1 juan users 49447 sep  6 15:28  Cert.Matr.Paula2024-2025.pdf
drwxr-xr-x 2 juan users  4096 feb 10  2024  Archivos
drwxr-xr-x 4 juan users  4096 sep  5 17:34 'Automatic Upload'
drwxr-xr-x 4 juan users  4096 sep  4 17:42  Backups
drwxr-xr-x 2 juan users  4096 feb  3  2024 'My Music'
drwxr-xr-x 6 juan users  4096 feb 24  2024 'My Pictures'
drwxr-xr-x 3 juan users  4096 sep  5 17:35 'My Videos'
drwxr-xr-x 2 juan users  4096 mar 23 12:20 'New Folder'
drwxr-xr-x 2 juan users  4096 sep  3 20:56 'Public Folder'
drwxr-xr-x 2 juan users  4096 feb 15  2024 'System Volume Information'

UID: 1000
GID: 100.
In theory they are user and Juan would have to give you access even though the gid is 100

**Comment by BlockyBlockling** [comment]
Did you try to open them as root?

**Comment by Git72job** [comment]
Si pero no deja.
Lo conseguí utilizando otro método a través de webtop
